src/Tribe/Views/V2/Utils/Separators.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Provides common View v2 utilities.
+ *
+ * @since   TBD
+ * @package Tribe\Events\Views\V2\Utils
+ */
+namespace Tribe\Events\Views\V2\Utils;
+
+use Tribe__Utils__Array as Arr;
+
+/**
+ * Class Utils Separators
+ * @since   TBD
+ * @package Tribe\Events\Views\V2\Utils
+ */
+class Separators {
+
+	/**
+	 * Determines if a given event from a list of events should have a month separator
+	 * for the List view template structure.
+	 *
+	 * @since  TBD
+	 *
+	 * @param  array       $events WP_Post or numeric ID for events
+	 * @param  WP_Post|int $event  Event we want to determine
+	 *
+	 * @return boolean
+	 */
+	public static function should_have_month( $events, $event ) {
+		if ( ! is_numeric( reset( $events ) ) ) {
+			$ids = wp_list_pluck( $events, 'ID' );
```

Comment:
This approach will break with a mix of IDs and events; it should be easy to refactor this code into an `array_map` like:
```php
$ids = array_map(static function($event){
    return is_numeric($event) ? $event : $event->ID;
}, $events);
```